spec/unit/room.spec.ts

```diff
@@ -2985,7 +3020,6 @@ describe("Room", function () {
 
             await emitPromise(room, ThreadEvent.Update);
             const threadResponse2Redaction = mkRedaction(threadResponse2);
-            await emitPromise(room, ThreadEvent.Update);
```

Comment:
I can't work out why this would have been here: why would we expect simply making the event to trigger an update?